delivery_recipe/business_continuity_plan.md

```diff
@@ -0,0 +1,134 @@
+
+* Respond to a disruptive incident (incident management)
+* Maintain delivery of critical activities during an incident (business continuity)
+* Return to "business as usual" (resumption and recovery)
+
+## Plan Remit
+
+The following services and functions are covered by this plan:
+
+* Project management and delivery
+* Service management to provide support for live projects
+
+## Plan Owner
+
+The plan is owned by the following people:
+
+* Steve Parks - CEO
+* Steve Hunton - COO
+
+These people are responsible for ensuring that the plan is maintained, exercised and updated in accordance with internal requirements for business continuity.
+
+## Plan Distribution
+
+The plan is made available online to all staff via our cookbook. It can be accessed from any device and access is not restricted. This makes it easy to access in the event of an emergency.
+
+## Plan Review
+
+The plan is reviewed annually (as a minimum). The next review date is planned for June 2017.
+
+## Plan Testing
+
+The plan is due to be tested before the next review in June 2017.
+
+# Plan Activation
+
+## Circumstances
+
+The plan will be activated in response to an incident causing significant disruption to normal service delivery, particularly in the delivery of projects or access to data. Examples of circumstances triggering activation of this plan include:
+
+* Loss of critical systems
+* Sudden loss of key staff or skills
+* Loss or theft of mobile devices (e.g. laptop, phone)
+* Loss, including suspected loss or unauthorised access, of data
+
+## Responsibility for Activation
+
+A member of our Business Continuity Team (BCT) will activate and initiate this plan:
+
+* Steve Parks, CEO
+* Steve Hunton, COO
+* Joe Baker, Lead Developer
+
+# Incident Management
+
+## Purpose of the Incident Management Phase
+
+* Protect vital assets e.g. equipment, data, reputation, clients
+* Ensure necessary communication takes place
+* Support the Business Continuity phase
+* Support the Recovery and Resumption phase
+
+## Actions to Protect Vital Assets
+
+The following actions will be taken to protect our vital assets:
+
+1. Whoever identifies the incident should record and report details of the incident. If possible, email these to hello@weareconvivio.com
+2. Inform members of the Business Continuity Team (see above)
+3. The BCT assess the impact of the incident to agree response and next steps
+4. The BCT team communicates the incident to other members of the team and customer as agreed in their next steps
+
+## Communication Actions
+
+In the event of an incident and this plan being initiated, the BCM team should be contacted. Other people within the team or within the client base may also be contacted depending on the details of the incident.
+
+# Incident Scenarios
+
+Following are a list of possible incident scenarios that could risk business continuity and the key steps to be taken in order to manage the risks.
+
```

Comment:
I would insert a section here for the benefit of others who don't know our business:

Loss of access to building
We operate as a 'free range' company meaning that our default state is that staff can work from anywhere. In the event of our office at the Gridiron not being available, or it being advisable not to congregate in the office (eg in the case of pandemic), this will not cause business disruption, even for extended periods.